stp_core/config.py

```diff
@@ -44,6 +44,10 @@
 ZMQ_CLIENT_QUEUE_SIZE = 100  # messages (0 - no limit)
 ZMQ_NODE_QUEUE_SIZE = 20000  # messages (0 - no limit)
 ZMQ_STASH_TO_NOT_CONNECTED_QUEUE_SIZE = 10000
+PENDING_CLIENT_MESSAGES_LIMIT = 100
```

Comment:
I think `PENDING_CLIENT_LIMIT` is a better name.

Reply:
I'm not sure because it sounds like a limit of clients. If it's ok, I'll change it.